src/main/scala/com/fortysevendeg/android/sqlite/resultset/SQLDroidResultSet.scala

```diff
@@ -0,0 +1,688 @@
+package com.fortysevendeg.android.sqlite.resultset
+
+import java.io._
+import java.math.BigDecimal
+import java.net.URL
+import java.sql._
+import java.text.SimpleDateFormat
+import java.util
+import java.util.Calendar
+
+import android.database.Cursor
+import com.fortysevendeg.android.sqlite._
+
+import scala.util.{Failure, Success, Try}
+import scala.{Array => SArray}
+
+class SQLDroidResultSet(
+  cursor: Cursor,
+  log: LogWrapper = new AndroidLogWrapper())
+  extends ResultSet
+  with NonEditableResultSet
+  with WrapperNotSupported {
+
+  private[this] var lastColumnRead: Option[Int] = None
+
+  private[this] def moveOp(f: => Boolean): Boolean =
+    f match {
+      case true =>
+        lastColumnRead = None
+        true
+      case false =>
+        false
+    }
+
+  private[this] def getOp[T](index: Int)(f: Int => T): T = {
+    lastColumnRead = Some(index)
+    f(index)
+  }
+
+  private[this] val dateFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss.S")
```

Comment:
Maybe we should create a DateUtils class/object with a helper method with this date format but allowing pass the format.

Can we use joda.Date?